usermods/AHT10_v2/usermod_aht10.h

```diff
@@ -0,0 +1,201 @@
+      temp.add(_lastLoopCheck);
+      temp.add(F(" / "));
+      temp.add(_lastStatus);
+#endif
+
+      JsonArray jsonTemp = user.createNestedArray(F("AHT Temperature"));
+      JsonArray jsonHumidity = user.createNestedArray(F("AHT Humidity"));
+      
+      if (_lastLoopCheck == 0)
+      {
+        // Before first run
+        jsonTemp.add(F("Not read yet"));
+        jsonHumidity.add(F("Not read yet"));
+        return;
+      }
+
+      if (_lastStatus != AHT10_SUCCESS)
+      {
+        jsonTemp.add(F("An error occurred"));
+        jsonHumidity.add(F("An error occurred"));
+        return;
+      }
+
+      jsonTemp.add(_lastTemperature);
+      jsonTemp.add(F("°C"));
+
+      jsonHumidity.add(_lastHumidity);
+      jsonHumidity.add(F("%"));
+    }
+
+    void addToConfig(JsonObject& root)
+    {
+      JsonObject top = root.createNestedObject(FPSTR(_name));
+      top[F("Enabled")] = _enabled;
+      top[F("I2CAddress")] = static_cast<uint8_t>(_i2cAddress);
+      top[F("SensorType")] = _ahtType;
+      top[F("CheckInterval")] = _checkInterval / 1000;
+      top[F("Decimals")] = log10f(_decimalFactor);
+
+      DEBUG_PRINTLN(F("AHT10 config saved."));
+    }
+
+    bool readFromConfig(JsonObject& root) override
+    {
+      // default settings values could be set here (or below using the 3-argument getJsonValue()) instead of in the class definition or constructor
+      // setting them inside readFromConfig() is slightly more robust, handling the rare but plausible use case of single value being missing after boot (e.g. if the cfg.json was manually edited and a value was removed)
+
+      JsonObject top = root[FPSTR(_name)];
+
+      bool configComplete = !top.isNull();
+      if (!configComplete)
+        return false;
+
+      configComplete &= getJsonValue(top["Enabled"], _enabled);
+      configComplete &= getJsonValue(top["I2CAddress"], _i2cAddress);
+      configComplete &= getJsonValue(top["CheckInterval"], _checkInterval);
+      if (configComplete)
+      {
+        if (1 <= _checkInterval && _checkInterval <= 600)
+          _checkInterval *= 1000;
+        else
+          // Invalid input
+          _checkInterval = 60000;
+      }
+
+      configComplete &= getJsonValue(top["Decimals"], _decimalFactor);
+      if (configComplete)
+      {
+        if (0 <= _decimalFactor && _decimalFactor <= 5)
+          _decimalFactor = pow10f(_decimalFactor);
+        else
+          // Invalid input
+          _decimalFactor = 100;
+      }
+
+      uint8_t tmpAhtType;
+      configComplete &= getJsonValue(top["SensorType"], tmpAhtType);
+       if (configComplete)
+      {
+        if (0 <= tmpAhtType && tmpAhtType <= 2)
+          _ahtType = static_cast<ASAIR_I2C_SENSOR>(tmpAhtType);
+        else
+          // Invalid input
+          _ahtType = 0;
+      }
+
+      if (_initDone)
+      {
+        // Reloading config
+        initializeAht();
+      }
+
+      _initDone = true;
+      return configComplete;
+    }
+};
```

Comment:
Please also implement `getId()` method. I know it is not mandatory ATM but may be required in future enhancements.